justfile

```diff
@@ -0,0 +1,37 @@
+IS_PROD := env_var_or_default("IS_PROD", "")
+COMPOSE_FILE := "-f docker-compose.yml " + if IS_PROD == "true" {""} else {"-f docker-compose.override.yml "}
+DC := "docker-compose " + COMPOSE_FILE
+RUN := DC + " run --rm app"
+set dotenv-load := false
```

Comment:
I feel like at this point `just` should just have a `dc` variant with all this built in 😂

Reply:
Seriously! I wish we could host a common `justfile` somewhere and import it here so we didn't have to keep repeating ourselves. I might end up doing a repo-sync similar to some other projects where the `justfile` is defined in one place and updated everywhere. I hate having to keep updating the dozen or so projects I work with!

Reply:
Drat lol: https://github.com/casey/just/issues/906